types/office-js/index.d.ts

```diff
@@ -11281,6 +11557,36 @@ declare namespace Office {
          *                 asyncResult, which is an Office.AsyncResult object.
          */
         addHandlerAsync(eventType: Office.EventType, handler: any, callback?: (result: AsyncResult<void>) => void): void;
+
+        /**
+         * Gets an attachment from a message or appointment and returns it as an Office.AttachmentContent object.
+         * 
+         * The getAttachmentContentAsync method gets the attachment with the specified identifier from the item. As a best practice, you should use 
+         * the identifier to retrieve an attachment in the same session that the attachmentIds were retrieved with the `getAttachmentsAsync` or 
+         * `item.attachments` call. In Outlook Web App and OWA for Devices, the attachment identifier is valid only within the same session. A session 
+         * is over when the user closes the app, or if the user starts composing an inline form and subsequently pops out the line form to continue 
```

Comment:
"starts composing an inline form and subsequently pops out the line form to continue" was confusing to me, but I'm not as familiar with Outlook vernacular. Is "form" the general term for message, meeting, and appointment? And is "inline" when I'm in the same window as my inbox?